tdrs-backend/tdpservice/settings/common.py

```diff
@@ -199,6 +200,7 @@ class Common(Configuration):
     # Sessions
     SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
     SESSION_COOKIE_HTTPONLY = True
+    SESSION_TIMEOUT = 30
```

Comment:
This is the default, so if it isn't defined in the appropriate environment settings file it will be 30 minutes.

Reply:
Look good! It might be helpful include the unit of measurement in the variable, something like `SESSION_TIMEOUT_MINUTES`, but that's a small suggestion and not something we have to wait on to merge this PR.